.travis/setup_binfmt_misc.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -ex
+
+echo ':arm:M::\x7fELF\x01\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x28\x00:\xff\xff\xff\xff\xff\xff\xff\x00\xff\xff\xff\xff\xff\xff\xff\xff\xfe\xff\xff\xff:/usr/local/bin/qemu-arm-static:' > /proc/sys/fs/binfmt_misc/register
```

Comment:
Yes, it seems that is possible. However I'm not sure whether that's actually needed though. We might not need `docker run` if we can already enter the image-name that Drone needs to use: http://readme.drone.io/build/build.html. Not sure whether we can use different images based on the build parameters. `privileged` is also available.